keps/sig-apps/3939-allow-replacement-when-fully-terminated/README.md

```diff
@@ -708,6 +708,13 @@ This section must be completed when targeting beta to a release.
 
 #### How can a rollout or rollback fail? Can it impact already running workloads?
 
+If a user has set `PodReplacementPolicy: Failed` or has PodFailurePolicy set, then
+rollbacking this feature would mean that replacing terminating pods would happen immediately.
```

Comment:
Can you expand more on what this can mean from a cluster admin perspective? What can happen to the cluster when rollback in those conditions happens? Example, increased failure rates of pods in jobs having that feature set, etc.

Reply:
Updated.